### ISSUE TITLE
Issue with reproducing the performance of the model presented in the paper

### ISSUE THREAD

**Issue by hwang9u** [Jiaxin-Ye/TIM-Net_SER#8]
Hello. Thank you for your excellent work.
I'm also very interested in speech emotion recognition!!

I am experiencing an issue with reproducing the performance of the model presented in the paper.
Would it be possible for you to share the results of your experiments, such as the loss curves and the results on each dataset? I would like to compare my experiments with yours.

Thank you very much. Have a great day!!

**Comment by Jiaxin-Ye** [comment]
Thanks for your attention! The result and model of each corpus is available and you can train your own model to compare with them. However, for the loss curves, we didn't save all curves in our experiments and it is hard for us to find all the curve data again. We apologize for any inconvenience caused to you. 
Furthermore, due to hardware and software differences and randomness and initialization, the reproducing results may differ from our results. Considering this situation, we provide our pretrained models to let you test. If you think this comparison is unfair, you can retrain all the baselines in your local environment for comparative experiments.

Thanks.

**Comment by OhadCohen97** [comment]
Hi!

I am also experiencing an issue reproducing the performance of the model presented in the paper.
How did you manage to achieve 92.08% accuracy? 
After training from scratch, I got average accuracy of 80% (seed=46). 

Thank you for your work.

**Comment by hwang9u** [comment]
Hi!
Nice to meet you :)
I had the same difficulty as you. I saw an email notification from you and
entered github, but I couldn't check your comment.
Where should I check?

Best regards,
Seonjoo Kim

2023년 7월 3일 (월) 오후 10:04, Ohad Cohen ***@***.***>님이 작성:
